homeassistant/components/vallox/__init__.py

```diff
@@ -291,3 +311,26 @@ async def async_handle(self, call: ServiceCall) -> None:
         # be observed by all parties involved.
         if result:
             await self._coordinator.async_request_refresh()
+
+
+class ValloxEntity(CoordinatorEntity[ValloxDataUpdateCoordinator]):
+    """Representation of a Vallox entity."""
+
+    def __init__(self, name: str, coordinator: ValloxDataUpdateCoordinator) -> None:
+        """Initialize a Vallox entity."""
+        super().__init__(coordinator)
+
+        self._device_uuid = self.coordinator.data.get_uuid()
+
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, str(self._device_uuid))},
+            manufacturer=DEFAULT_NAME,
+            model=self.coordinator.data.get_model(),
+            name=name,
+            sw_version=self.coordinator.data.get_sw_version(),
+        )
+
+        if self.coordinator.config_entry is not None:
```

Comment:
I don't think its possible for this to be None

Reply:
Suggested changes https://github.com/bdraco/home-assistant/commit/60066ca069a919415b69755fd92f0217857316ef

Reply:
Thanks a lot for that! I pulled this in and tested it locally. Works.